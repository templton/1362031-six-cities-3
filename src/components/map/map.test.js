import React from "react";
import renderer from "react-test-renderer";
import Map from "./map.jsx";

const cityCord = [52.38333, 4.9];
const placeCords = [[52.3909553943508, 4.91309666406198]];

describe(`Map renderer`, () => {

  it(`simple renderer`, () => {
    const tree = renderer
      .create(<Map cityCord={cityCord} placeCords={placeCords}/>, {
        createNodeMock: () => {
          return document.createElement(`div`);
        }
      }).toJSON();

    expect(tree).toMatchSnapshot();
  });

});
