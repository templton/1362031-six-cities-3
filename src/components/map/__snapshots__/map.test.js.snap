// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map renderer simple renderer 1`] = `
<section
  className="cities__map map"
>
  <div
    id="map"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</section>
`;
